src/color/background.json

```diff
@@ -1,46 +1,276 @@
 {
   "color": {
     "background": {
+      "100": {
+        "default": {
+          "value": "{color.base.white.value}",
+          "public": true,
+          "default": true,
+          "neutral": true,
```

Comment:
initially the `neutral` key was reserved for "neutral" colors. E.g. grayscale. I see that this has been incorrectly used in multiple places and we aren't really using it as a filter option. 

Please remove from these tokens.